src/mixins/portfolio.js

```diff
@@ -405,8 +508,10 @@ export const createPorfolioMixin = ({
     changeTab(tab) {
       if (!tabOptions[tab]) return;
       this.syncRouteForTab(tab);
+      this.nftTypeFilter = NFT_TYPE_OPTIONS.ALL;
```

Comment:
Seems more intuitive
```suggestion
      this.nftTypeFilter = NFT_TYPE_FITLER_OPTIONS.ALL;
```